src/app/donation-start/donation-start-form/donation-start-form.component.html

```diff
@@ -18,16 +18,16 @@
         @if (campaign.matchFundsRemaining > 0) {
           <p class="donation_highlight">Match funds are currently available!</p>
         }
-        <div class="donationInfoWrapper">
-          <div>
-            <p class="b-rt-0 b-m-0">You are supporting:</p>
-            <p class="b-rt-0 b-m-0 b-bold">{{ campaign.charity.name }}</p>
-          </div>
-          <div>
-            <p class="b-rt-0 b-m-0">for:</p>
-            <p class="b-rt-0 b-m-0 b-bold">{{ campaign.title }}</p>
-          </div>
-        </div>
+        <h1 class="donationInfoWrapper">
```

Comment:
I started with a separate visually hidden `h1`, but ultimately this is doing that job already. I think slightly clunky markup to allow for varying styles within the title is a smaller cost than maintaining two different elements.